scripts/install-windows-prereqs.ps1

```diff
@@ -449,26 +497,31 @@ function Install-DCAPDrivers {
     $TEMP_NUGET_DIR = "$PACKAGES_DIRECTORY\Azure_DCAP_Client_nupkg"
     New-Directory -Path $OE_NUGET_DIR -RemoveExisting
     New-Directory -Path $TEMP_NUGET_DIR -RemoveExisting
-    $nupkgDir = Get-Item "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\nupkg"
+    $nupkgDir = Get-Item "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\nuget"
     if(!$nupkgDir) {
         Throw "Cannot find the Intel DCAP nupkg directory"
     }
     if($nupkgDir.Count -gt 1) {
-        Throw "Multiple Intel DCAP nupkg directories found"
+        Throw "Multiple Intel DCAP nuget directories found"
     }
     Copy-Item -Recurse -Force "$nupkgDir\*" $TEMP_NUGET_DIR
     Copy-Item $PACKAGES['azure_dcap_client_nupkg']['local_file'] -Destination $TEMP_NUGET_DIR -Force
-    & "$PACKAGES_DIRECTORY\nuget.exe" install 'EnclaveCommonAPI' -Source "$TEMP_NUGET_DIR" -OutputDirectory "$OE_NUGET_DIR" -ExcludeVersion
-    if($LASTEXITCODE -ne 0) {
-        Throw "Failed to install nuget EnclaveCommonAPI"
+
+    # Note: the ordering of nuget installs below is important to preserve here until the issue with the EnclaveCommonAPI nuget package gets fixed.
+    if ($WithAzureDCAPClient -eq $true)
+    {
+        & "$PACKAGES_DIRECTORY\nuget.exe" install 'Microsoft.Azure.DCAP.Client' -Source "$TEMP_NUGET_DIR;nuget.org" -OutputDirectory "$OE_NUGET_DIR" -ExcludeVersion
+        if($LASTEXITCODE -ne 0) {
+            Throw "Failed to install nuget EnclaveCommonAPI"
+        }
     }
-    & "$PACKAGES_DIRECTORY\nuget.exe" install 'DCAP_Components' -Source "$TEMP_NUGET_DIR" -OutputDirectory "$OE_NUGET_DIR" -ExcludeVersion
+    & "$PACKAGES_DIRECTORY\nuget.exe" install 'DCAP_Components' -Source "$TEMP_NUGET_DIR;nuget.org" -OutputDirectory "$OE_NUGET_DIR" -ExcludeVersion
     if($LASTEXITCODE -ne 0) {
-        Throw "Failed to install nuget DCAP_Components"
+        Throw "Failed to install nuget EnclaveCommonAPI"
     }
-    & "$PACKAGES_DIRECTORY\nuget.exe" install 'Microsoft.Azure.DCAP.Client' -Source "$TEMP_NUGET_DIR;nuget.org" -OutputDirectory "$OE_NUGET_DIR" -ExcludeVersion
+    & "$PACKAGES_DIRECTORY\nuget.exe" install 'EnclaveCommonAPI' -Source "$TEMP_NUGET_DIR;nuget.org" -OutputDirectory "$OE_NUGET_DIR" -ExcludeVersion
     if($LASTEXITCODE -ne 0) {
-        Throw "Failed to install nuget Microsoft.Azure.DCAP.Client"
+        Throw "Failed to install nuget EnclaveCommonAPI"
     }
 
     New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Services\sgx_lc_msr\Parameters" -Name "SGX_Launch_Config_Optin" -Value 1 -PropertyType DWORD -Force
```

Comment:
Could we please add a comment here?

Reply:
What kind of comment do you mean? Above (line 510) there is a comment that says: "# Note: the ordering of nuget installs below is important to preserve here until the issue with the EnclaveCommonAPI nuget package gets fixed."

Reply:
This is line 527 where the registry is getting updated.  Not sure what setting a value of 1 to SGX_Launch_Config_Optin does.

Reply:
Oh I see what you mean! I'm not sure :) I think it's how we enable certain parts of the FLC driver. I should find out where this is documented and add a link to it!

Reply:
It's necessary as an administrator action to enable the FLC driver to run on Windows. See https://download.01.org/intel-sgx/dcap-1.2/windows/docs/Intel_SGX_DCAP_Windows_SW_Installation_Guide.pdf

Reply:
I've included a link to this document in a comment in the script. Thanks @CodeMonkeyLeet !